_IBC/joltify-cosmoshub.json

```diff
@@ -0,0 +1,31 @@
+{
+    "$schema": "../ibc_data.schema.json",
+    "chain_1": {
+      "chain_name": "joltify",
```

Comment:
IBC connection registrations must order chain 1 and chain 2 is alphabetical order. Please make joltify chain 2 when paired with cosmoshub, and reverse the order of the file name as well (should be 'cosmoshub-joltify.json')